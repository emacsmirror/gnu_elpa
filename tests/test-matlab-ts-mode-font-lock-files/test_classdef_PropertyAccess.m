% -*- matlab-ts -*-
classdef test_classdef_PropertyAccess
   properties (GetAccess = {?ClassA, ?ClassB})
      Prop1
   end
   properties (Access = ?ClassC)
      Prop2
   end
end
