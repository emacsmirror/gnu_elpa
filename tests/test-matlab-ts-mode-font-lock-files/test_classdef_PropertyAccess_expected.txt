c ccc ccccccccc ccc
kkkkkkkk ffffffffffffffffffffffffffff
   kkkkkkkkkk bttttttttt d bDddddddD Dddddddbb
      PPPPP
   kkk
   kkkkkkkkkk btttttt d Dddddddb
      PPPPP
   kkk
kkk
